templates/3dsmax/scripts/3dsmax.ps1

```diff
@@ -5,7 +5,7 @@ param (
     [string]$sceneFile,
     [int]$nodeCount = 1,
     [switch]$dr,
-    [string]$renderer = "vray",
+    [string]$renderer = "vray5",
```

Comment:
from my understading, the `$renderer` variable is always set in the job command lines, and comes from the job "renderer" parameter
for instance
```
"commandLine": "powershell.exe 3dsmax.ps1 -maxVersion \\\"[parameters('maxVersion')]\\\" -start {0} -end {0} -outputName \\\"images\\[parameters('outputName')]\\\" -sceneFile \\\"[parameters('sceneFile')]\\\" -pathFile \\\"[parameters('pathFile')]\\\" -renderer [parameters('renderer')] -preRenderScript \\\"[parameters('preRenderScript')]\\\" -camera \\\"[parameters('camera')]\\\" -additionalArgs \\\" [parameters('additionalArgs')]\\\" -irradianceMap \\\"[parameters('irradianceMapFile')]\\\" -renderPresetFile \\\"[parameters('renderPresetFile')]\\\"",
```

In the job template, the allowed values are either "arnold" or "vray", never "vray5"

Reply:
yes this was a bit confusing and overloaded in prior templates - some templates, like those for VRay DR - have both a renderer parameter (set to vray) and a vrayRenderer param (set to vrayAdv, VrayRT). Sometimes the value of vrayRenderer was passed in as the value to the call to "3dsmax.ps1 -renderer $value",